iosApp/iosApp/ui/textFields.swift

```diff
@@ -53,7 +55,7 @@ struct PasswordTextField: View {
     @Binding var isPasswordVisible: Bool
     
     var body: some View {
-        EditText(input: $password, topLabel: "Password", isSecure: !isPasswordVisible)
+        EditText(input: $password, topLabel: "Wachtwoord", isSecure: !isPasswordVisible)
```

Comment:
Hardcoded text

Reply:
Yarin made a pull request with a string file shared between android and ios, waiting for that pr so I can change the hardcoded strings in ios